src/muse/data/example/default/settings.toml

```diff
@@ -60,7 +60,8 @@ constraints = [
     "max_capacity_expansion",
     "demand",
     "search_space",
-    "minimum_service"
+    "minimum_service",
+    "demand_limitting_capacity"
```

Comment:
Just a note that this change will also change the settings files for the tutorial models once `generate_models.py` is run (although this won't change the results), so may as well do that now

Reply:
Didn't I change the tutorials files as well? My fault if I didn't...